coralproject/talk
# Webpack Build Performance Improvements

## What does this PR do?

- Applied some tweaks recommended by Slack (ref: https://slack.engineering/keep-webpack-fast-a-field-guide-for-better-build-performance-f56a5995e8f1) for optimizing webpack build speed

## How do I test this PR?

1. Run `rm -rf node_modules && yarn cache clean && yarn && yarn build`
2. See it run!
